bundles/org.openhab.binding.salus/src/main/java/org/openhab/binding/salus/internal/handler/CloudBridgeHandler.java

```diff
@@ -0,0 +1,291 @@
+        try {
+            x.findDevices();
+        } catch (Exception ex) {
+            var msg = "Cannot connect to Salus Cloud! Probably username/password mismatch!";
+            logger.error(msg, ex);
+            updateStatus(OFFLINE, CONFIGURATION_ERROR, msg + " " + ex.getMessage());
+            return;
+        }
+        this.devicePropertiesCache = Caffeine.newBuilder().maximumSize(10_000)
+                .expireAfterWrite(Duration.ofSeconds(propertiesRefreshInterval))
+                .refreshAfterWrite(Duration.ofSeconds(propertiesRefreshInterval)).build(this::loadPropertiesForDevice);
+        var scheduledPool = ThreadPoolManager.getScheduledPool("Salus");
+        this.scheduledFuture = scheduledPool.scheduleWithFixedDelay(this::refreshCloudDevices, refreshInterval * 2,
+                refreshInterval, SECONDS);
+
+        // done
+        updateStatus(ONLINE);
+    }
+
+    private void loadConfigs() {
+        var config = this.getConfig();
+        username = (String) config.get("username");
+        password = ((String) config.get("password")).toCharArray();
+        url = (String) config.get("url");
+        if ("".equals(url)) {
+            url = "https://eu.salusconnect.io";
+        }
+        refreshInterval = ((BigDecimal) config.get("refreshInterval")).longValue();
+        if (refreshInterval <= 0) {
+            this.refreshInterval = 30;
+        }
+        propertiesRefreshInterval = ((BigDecimal) config.get("propertiesRefreshInterval")).longValue();
+        if (propertiesRefreshInterval <= 0) {
+            this.propertiesRefreshInterval = 5;
+        }
+    }
+
+    private void refreshCloudDevices() {
+        logger.debug("Refreshing devices from CloudBridgeHandler");
+
+        if (!(thing instanceof Bridge bridge)) {
+            logger.debug("No bridge, refresh cancelled");
+            return;
+        }
+        var things = bridge.getThings();
+        for (var thing : things) {
+            if (!thing.isEnabled()) {
+                logger.debug("Thing {} is disabled, refresh cancelled", thing.getUID());
+                continue;
+            }
+            var handler = thing.getHandler();
+            if (handler == null) {
+                logger.debug("No handler for thing {} refresh cancelled", thing.getUID());
+                continue;
+            }
+            var channels = thing.getChannels();
+            for (var channel : channels) {
+                handler.handleCommand(channel.getUID(), REFRESH);
+            }
+        }
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        // no commands in this bridge
+        logger.debug("Bridge does not support any commands to any channels. channelUID={}, command={}", channelUID,
+                command);
+    }
+
+    @Override
+    public void dispose() {
+        var f = scheduledFuture;
```

Comment:
Please use meaningfull var namens, please check if there are more.

Reply:
I do not agree here (but changed). `f` is a short living var that can have (in my opinion) such a name.